### PR TITLE
feature/PLU-4180-sort-api-resource-operations-ecs

### PR DESCRIPTION
**Pull request by alexandrtspl**
* sorted api resource operations by ecs

Based on AlphabeticallySortedArrayKeysSniff with a little changes. 

image

### CODE REVIEW

File: src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+use EonX\EasyQuality\Output\Printer;
+use Error;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\ParserFactory;
+
+final class SortedApiResourceOperationsKeysSniff implements Sniff
```
Comment (alexndlm) [inline src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php:16]: WDYT, about extracting duplicated `AlphabeticallySortedArrayKeysSniff` code to `AbstractSortedArrayKeysSniff`?

**Review by alexndlm — CHANGES_REQUESTED** [review]

File: src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php (hunk 1 above)
Comment (alexandrtspl) [reply src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php:16]: I don't mind, but I doubt it

**Review by alexndlm — APPROVED** [review]

File: src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php [hunk 2]
```diff
@@ -0,0 +1,319 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyQuality\Sniffs\Arrays;
```
Comment (roman-eonx) [inline src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php:5]: Not the best place, as we are not dealing with arrays here. Maybe `EonX\EasyQuality\Sniffs\Attributes\`?

File: src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php (hunk 1 above)
Comment (roman-eonx) [inline src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php:16]: Please rename to `SortedApiResourceOperationKeysSniff`.

**Review by roman-eonx — CHANGES_REQUESTED** [review]

File: src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php (hunk 1 above)
Comment (alexandrtspl) [reply src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php]: Done

File: src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php (hunk 2 above)
Comment (alexandrtspl) [reply src/Sniffs/Arrays/SortedApiResourceOperationsKeysSniff.php]: Moved

**Review by alexndlm — APPROVED** [review]

**Review by roman-eonx — APPROVED** [review]

**Pull request merged**